sample/project/plugins.sbt

```diff
@@ -5,6 +5,8 @@ logLevel := Level.Warn
 resolvers += "Typesafe Releases Repository" at "http://repo.typesafe.com/typesafe/releases/"
 
 // Use the Play sbt plugin for Play projects
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.0")
 
-addSbtPlugin("com.linkedin" % "play-plugins-testng" % "2.4.0")
+addSbtPlugin("de.johoop" % "sbt-testng-plugin" % "3.1.1")
```

Comment:
Why is this needed?

Reply:
Okay, turns out I don't need to import the TestNG plugin directly.